moveit_simple/src/moveit_simple.cpp

```diff
@@ -659,6 +788,46 @@ void OnlineRobot::execute(const std::string traj_name, bool collision_check)
 
 
 
+void OnlineRobot::reconfigureRequest(moveit_simple_dynamic_reconfigure_Config &config, uint32_t level)
+{
+  params_.fromConfig(config);
+  if (params_.speed_percent > 0.0)
+  {
+    setSpeedModifier(params_.speed_percent);
+  }
+}
+
+
+
+void OnlineRobot::setSpeedModifier(double speed_percent)
+{
+  if (speed_percent <= 1.0 && speed_percent > 0.0)
+  {
+    speed_modifier_ = speed_percent;
+  }
+  else if((speed_percent > 1.0))  
+  {
+    speed_modifier_ = max_speed;
```

Comment:
Don't declare constant member varriables unless they are used in multiple methods.  Otherwise just create them in the method in which they are used.